zebra-chain/src/types.rs

```diff
@@ -36,9 +38,12 @@ impl Arbitrary for BlockHeight {
 ///
 /// # Invariants
 ///
-/// Users should not construct a `LockTime` with `BlockHeight` greater than or
-/// equal to `500_000_000` or a timestamp before 4 November 1985 (Unix timestamp
-/// less than `500_000_000`).
+/// Users should not construct a `LockTime` with:
+///   - a `BlockHeight` greater than MAX_BLOCK_HEIGHT,
+///   - a timestamp before 6 November 1985
+///     (Unix timestamp less than MIN_LOCK_TIMESTAMP), or
+///   - a timestamp after 6 February 2106
```

Comment:
This advice isn't correct for all time zones, so it would be better to say:
```suggestion
///   - a timestamp after 5 February 2106
```

Reply:
(Specifically, the rollover time is early in the morning UTC 7 February , which is still 6 February in some time zones. Aren't time zones fun?)

Reply:
I'd like to say "approximately", but maybe it's clear already that the dates are approximate.